### ISSUE TITLE
Screen reader is not announcing the search results information after entering valid or invalid data for the 'search docs' edit filed present in the Search dialog in Windows.

### ISSUE THREAD

**Issue by msftedad** [readthedocs/readthedocs.org#12097]
## Details

* Read the Docs project URL: https://docs.citusdata.com/en/stable/
* Build URL (if applicable): NA
* Read the Docs username (if applicable): NA

**Repro steps:**

1. Open URL "https://docs.citusdata.com/en/stable/" in Edge browser.
2. Press tab key to reach the 'stable read the docs' control present in the bottom right corner of the page and activate it using Enter key.
3. Press tab key to navigate to search docs edit field. Search dialog will appear.
4. Verify whether Screen reader is announcing the search results information after entering valid or invalid data for the 'search docs' edit filed or not.

## Expected Result

Screen reader should announce the status of search results information after entering valid or invalid data for the 'search docs' edit filed.
Ex: Screen reader should announce as "Showing 1 to 4 of 4 records." or 'showing 0 to 0 of 0 records'.

## Actual Result

Screen reader is not announcing the status of search results information after entering valid or invalid data for the 'search docs' edit filed. Screen reader is remaining silent upon receiving the search results.

Observation: Screen readers (Narrator, JAWS, NVDA) is not announcing the displayed results.

https://github.com/user-attachments/assets/db325973-d16c-4a6d-aac6-6dc4dd49d691

<img width="898" alt="Image" src="https://github.com/user-attachments/assets/1969e303-773d-4315-bf69-441c99db7ba4" />
